docs/content/1.index.md

```diff
@@ -186,7 +186,10 @@ Then in your `nuxt.config.ts`, set the `classSuffix` option to an empty string:
 
 ```ts [nuxt.config.ts]
 export default defineNuxtConfig({
-  modules: ['@nuxtjs/color-mode'],
+  modules: [
+    '@nuxtjs/color-mode',
+    // '@nuxtjs/tailwindcss' // specify after
```

Comment:
ideally we wouldn't have to specify an order of these modules.

raised https://github.com/nuxt-modules/tailwindcss/issues/826 to discuss and resolve upstream in tailwindcss module

Reply:
@danielroe I've updated the docs to *not* have a dedicated section for integration with `@nuxtjs/tailwindcss` as there is nothing required to do anymore. The hook ordering we'll track in the tailwind module!